### PR TITLE
Refactors recovery component and adds to story book

### PR DESCRIPTION
**Pull request by Moejoe90**

### CODE REVIEW

File: app/components/wallet-selector/wallet-recover/recovery/RecoverWallet.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+import { RecoveryProcessesComponent } from "./RecoverWallet";
+
+storiesOf("RecoveryProcesses", module)
+  .add("Enter Seed", () => <RecoveryProcessesComponent submitForm={() => {}} />)
```
Comment (desfero) [inline app/components/wallet-selector/wallet-recover/recovery/RecoverWallet.stories.tsx]: @Moejoe90 maybe use here addon-actions to make storybook more interactive `submitForm(action("submitForm"))`? Same on line `9`

File: app/components/wallet-selector/wallet-recover/recovery/RecoverWallet.tsx [hunk 2]
```diff
@@ -2,55 +2,53 @@ import * as React from "react";
 import { FormattedMessage } from "react-intl-phraseapp";
 import { Link } from "react-router-dom";
 import { Col, Row } from "reactstrap";
-import { compose } from "redux";
+import { branch, compose, renderComponent } from "recompose";
```
Comment (desfero) [inline app/components/wallet-selector/wallet-recover/recovery/RecoverWallet.tsx]: +1

File: app/components/wallet-selector/wallet-recover/recovery/RecoverWallet.tsx [hunk 3]
```diff
@@ -83,26 +81,26 @@ export class RecoveryProcessesComponent extends React.Component<
...
-  return <RecoveryProcessesComponent {...props} />;
-};
-
-export const RecoverWallet = compose<React.SFC>(
-  appConnect<any, IDispatchProps>({
+const RecoverWallet = compose<IDispatchProps, IStateProps>(
+  appConnect<IStateProps, IDispatchProps>({
     stateToProps: s => ({
-      isMessageSigning: s.walletSelector.isMessageSigning,
+      isMessageSigning: selectIsMessageSigning(s),
```
Comment (desfero) [inline app/components/wallet-selector/wallet-recover/recovery/RecoverWallet.tsx]: +1

File: app/components/wallet-selector/wallet-recover/recovery/RecoverWallet.stories.tsx (hunk 1 above)
Comment (Moejoe90) [reply app/components/wallet-selector/wallet-recover/recovery/RecoverWallet.stories.tsx]: cool idea

**Pull request merged**